.github/workflows/regression_tests.yml

```diff
@@ -26,6 +26,14 @@ jobs:
           python -m pip install --upgrade pip
           python -m pip install -r requirements.txt
           pip freeze | egrep 'iree|tensorflow'
+          python -m pip install flake8 pytest
+          
+      - name: Lint with flake8
```

Comment:
Lint checks should be added in a separate action that runs on pull requests. This action is just for daily regression testing.